src/components/WorkOrder/InventoryItems.js

```diff
@@ -63,24 +78,39 @@ class InventoryItems extends Component {
   render() {
     return (
       <div>
-        <Header icon={faWrench} title="Add Inventory Items" />
+        <Header
+          icon={faWrench}
+          title={`${this.renderInventoryFormVerb()} Inventory Item`}
+        />
 
-        {this.state.isSubmitted && (
-          <SuccessMessage formType="Inventory Item" formVerb="adde" />
-        )}
+        {this.state.isSubmitted &&
+          (this.state.isEditable ? (
+            <SuccessMessage formType="Inventory Item" formVerb="edite" />
```

Comment:
I know you're just following a pattern that I probably established but I'm wondering if it would be better to not assume the formVerb is going to end in `d` and just have the `formVerb` prop be the full spelled out word.

Reply:
I agree - it looks a little weird this way. I changed e `SuccessMessage` component and updated the verbs each time it is used.